### ISSUE TITLE
Syncthing suddenly started crashing on activation

### ISSUE THREAD

**Issue by T12z** [Catfriend1/syncthing-android#602]
It is the actual syncthing service that is crashing, not the GUI-wrapper. I know, this here is about the GUI part, but maybe anyone has had this before while using the fork-wrapper.

So, I have no idea what led to this, all I've got is the log, finishing with:

> 17:41:58W/SyncthingRunnable exit reason = exitNoUpgradeAvailable. Another Syncthing instance may be already running.
> 17:41:58I/SyncthingService onServiceStateChange: from ERROR to DISABLED
> 17:41:58I/Util     runShellCommandGetOutput: Exited with code 0
> 17:41:58W/ApiRequest Request to https://127.0.0.1:8384/rest/system/shutdown failed, java.net.ConnectException: Failed to connect to /127.0.0.1:8384

long log: [17_41_58W_Synct.txt](https://github.com/Catfriend1/syncthing-android/files/4023725/17_41_58W_Synct.txt)

Never had any relevant issues before and could not see, having (visible) system changes in the last 24h.

### Version Information
    App Version: 1.3.2.2
    Syncthing Version: v1.? (the one currently distributed on fdroid)
    Android Version: Android 9
    Device manufacturer: Sony
    Device model: XZ1 / G8441

### Android Log
    (on request)

**Comment by Catfriend1** [comment]
Hi,

This is a crash in the native's go code. Please reach out to the forum for upstream dev's support as it has nothing to do with the wrapper. ( forum.syncthing.net )

Thank you.

**Comment by T12z** [comment]
Thanks!
I had a bit of a stroll through the topics of the https://forum.syncthing.net/c/support/android sub-forum, plenty of really odd happenings ... don't want be you ;)
So, it is always best to grab the adb log ... and just when I did that -- no crash, damn you Heisen-bug!

Now the only thing that I noticed in the log: "Stored folder metadata for "repo" is 750h47m22.950158278s old; recalculating", which is practically impossible, but I guess a local bug, maybe aging SD-Card.

thanks for your quick heads-up, though closing for *not-reproducible*
